lib/requester/core.js

```diff
@@ -208,6 +208,7 @@ module.exports = {
      * @param defaultOpts.followOriginalHttpMethod
      * @param defaultOpts.maxRedirects
      * @param defaultOpts.removeRefererHeaderOnRedirect
+     * @param defaultOpts.time
```

Comment:
requires a more explicit name for our requester option which maps to postman-request option.
e.g removeRefererHeaderOnRedirect -> removeRefererHeader.